src/biotite/structure/io/pdbx/convert.py

```diff
@@ -1658,9 +1665,16 @@ def _apply_transformations(structure, transformation_dict, operations):
             coord = matrix_rotate(coord, rotation_matrix)
             # Translate
             coord += translation_vector
+
+        assembly_transform_ids.append(
+            np.full(len(structure), "-".join(list(operation)))
+        )
```

Comment:
For performance reasons, I think it would be better to append single values here and repeat the array below.

```suggestion
        sym_ids.append("-".join(list(operation)))
```